### DISCUSSION TITLE
Initial Setup Questions

### DISCUSSION THREAD

**Discussion by um1chc5** [cungminh2710/coderum.dev#32] [Q&A] [answered]
**Dạ em muốn hỏi một số vấn đề liên quan đến khởi tạo app:**
- `DATABASE_URL`, `GITHUB_CLIENT_ID` & `GITHUB_CLIENT_SECRET` (nếu cần) hiện tại cần lấy ở đâu ạ.
- Hiện tại đã có wireframe/mockup design cho dự án này chưa ạ, và tham gia như thế nào được ạ.
-------
- Where can I currently get the `DATABASE_URL`, `GITHUB_CLIENT_ID` & `GITHUB_CLIENT_SECRET` (if needed)?
- Is there already a wireframe/mockup design for this project, and how can I participate?

**Comment by cungminh2710** [comment, marked as answer]
Good question An @um1chc5, the designer is working on the mockup and wireframe at the moment. In the meantime, you can start contributing by improving our design system on Storybook first - [ui.coderum.dev](https://ui.coderum.dev/). Here are some issues you can work on https://github.com/cungminh2710/coderum.dev/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22

In a week, I will add detailed instructions on how to contribute to the `Coderum` folder. Stay tuned!
